tests/util/test_traceutils.py

```diff
@@ -117,3 +118,13 @@ def test_is_service_or_endpoint_ignored() -> None:
 
     # don't ignore other services
     assert not is_service_or_endpoint_ignored("service3")
+
+
+def test_get_operation_specifier() -> None:
+    test_redis = "redis"
+    response_redis = get_operation_specifier(test_redis)
+    assert response_redis == "command"
+
+    test_dynamodb = "dynamodb"
+    response_dynamodb = get_operation_specifier(test_dynamodb)
+    assert response_dynamodb == "op"
```

Comment:
Nice work on this UT. I suggest you use parametrized tests for it; then, we can easily extend any other service we want to use the FUP feature.
```suggestion
@pytest.mark.parametrize(
        "span_name, expected_result",
        [
            ("something", None),
            ("redis", "command"),
            ("dynamodb", "op"),
        ],
    )
def test_get_operation_specifier(span_name: str, expected_result: Optional[str]) -> None:
    response = get_operation_specifier(span_name)
    assert response == expected_result
```